LICENSE

```diff
@@ -1,5 +1,5 @@
  Copyright (c) 2014 Docker, Inc.
- 
+
```

Comment:
Hah, I hadn't noticed that our entire `LICENSE` file is indented with a single space.  We should probably just strip all the leading whitespace in this whole file.